examples/stable-diffusion/train_text_to_image_sdxl.py

```diff
@@ -1175,8 +1181,6 @@ def compute_time_ids(original_size, crops_coords_top_left):
             if accelerator.sync_gradients:
                 progress_bar.update(1)
                 global_step += 1
```

Comment:
train_loss = torch.tensor(0, device='hpu')

Reply:
I put this at the end of step.